server/views/partials/layout.njk

```diff
@@ -53,6 +54,16 @@
 
 {% block header %}
   {% include "./header.njk" %}
+
+  {{ mojPrimaryNavigation({
```

Comment:
Although you've said in MOJ projects this goes above the phase banner, I think we should follow the GOV.UK design system's "must" [here](https://design-system.service.gov.uk/components/phase-banner/#how-it-works) and put it below:

> Your banner must be directly under the black GOV.UK header and colour bar.

Reply:
Isn't that where it is at the moment? I noticed yesterday we're using a HMPPS header variant that doesn't have the colour bar, but at the moment the navigation is directly below the black header

Reply:
Sorry if it wasn't clear, I was referring to the phase banner above, so black header -> phase banner -> navigation

Reply:
Ah yes. I'm not sure if we can do this without hacking the CSS or replacing `block content` with `block postNavigationContent` in each view though, which is similar to something you weren't keen on before 🤔

Reply:
Could we add a new block to the `beforeContent` block after the `govukPhaseBanner` and override it on pages that we don't want it on (which may just be the initial dashboard page at the moment)?

Reply:
like we do for the `backLink`

Reply:
Yep - it just needs some CSS hacking so that the navigation is full width but I'm working on reusing the CSS from the dashboard